RELEASE_PROCESS.md

```diff
@@ -154,7 +154,8 @@ https://buildkite.com/chef/chef-chef-server-master-habitat-build
 ### Building and Releasing the Release
 
 1. Select a version from the `current` channel that you wish to promote to `stable`. Make sure that this version has gone through the upgrade testing.
-1. Use expeditor to promote the build.  The expeditor command is of the form:
+2. Make sure you are part of the slack group `release-promoters`.
```

Comment:
can we include some directions on how to do that? or else include this as a FAQ.

Reply:
Ya that would be good. Thanks for the input